### PR TITLE
Test/controllers specs

### PR DESCRIPTION
**Pull request by blessedjasonmwanza**
## Test/controllers specs
In this TDD update;
- Install and configure ```Rspec``` and ```rails-controller-testing``` (required for template rendering testing)
- Create test cases that check to ensure ```pages```, ```users ```, ```posts``` controllers meets the following test cases;
  - If the response status was correct.
  - If a correct template was rendered.
  - If the response body includes correct placeholder text.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,3 +29,7 @@ This README documents steps necessary to get the application up and running.
 
 * Run project
   - Run ```rails s``` and visit [http://localhost:3000/](http://localhost:3000/) in your browser!
+
+* Tests
+ - Run (*ensure your postgreSQL is running as well*)
+    - ```rspec```
```
Comment (FrederickMih) [inline README.md]: - [x] Kindly document your README file using the [Microverse README template](https://github.com/microverseinc/readme-template). Include all necessary headers and customize the content to match your project. By doing this, your README file will become more descriptive and professional. Also, note that **Hello Rails app** is not the title of this app. This is a **Blog App**

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @blessedjasonmwanza , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


# To Highlights

- GitHub workflow has been followed :heavy_check_mark: 
- Linters checks are OK :checkered_flag: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (blessedjasonmwanza) [reply README.md]: Hi @FrederickMih , Thanks so much for your review. On ti making the requested changes right away.

Comment (blessedjasonmwanza) [reply README.md]: Hi @FrederickMih , Thank you so much for your review once more. I realized I was too excited that I resumed work from the initial hello test app. I have made the required changes and appreciate your professional review.

> HappyCoding

**Review by keneogu — APPROVED** [review]
> ## Hi @blessedjasonmwanza,
> Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
> 
> ## Congratulations!     
> Cheers and Happy coding!clapclapclap
> 
> image [ ](https://camo.githubusercontent.com/2366be437d35bce33dd123620530e9131a8ac147fe960e76a36614dc703e8e22/68747470733a2f2f692e70696e696d672e636f6d2f6f726967696e616c732f37372f30652f62302f37373065623035373434393538326334343637326232643264316363333962362e676966) image [ ](https://camo.githubusercontent.com/2366be437d35bce33dd123620530e9131a8ac147fe960e76a36614dc703e8e22/68747470733a2f2f692e70696e696d672e636f6d2f6f726967696e616c732f37372f30652f62302f37373065623035373434393538326334343637326232643264316363333962362e676966)
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**